database/factories/CommentFactory.php

```diff
@@ -7,7 +7,7 @@
 $factory->define(App\Models\Comment::class, function (Faker $faker) {
     return [
         'content'    => $faker->text,
-        'anon'       => $faker->randomNumber(),
+        'anon'       => (int) $faker->boolean(),
```

Comment:
@HDVinnie Can you confirm that this is supposed to be a boolean value?

Reply:
boolean is correct